### PR TITLE
cmake: add French translation

### PR DESCRIPTION
**Pull request by Neluji**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known): _translation from english_**

### CODE REVIEW

File: pages.fr/common/cmake.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# cmake
+
+> Système de construction logicielle multiplateforme, qui permet de générer des recettes de construction pour les systèmes de construction natifs.
+> Plus d'informations : <https://cmake.org/cmake/help/latest/manual/cmake.1.html>.
+
+- Générer une recette de construction `CMakeLists.txt` depuis le répertoire d'un projet :
```
Comment (navarroaxel) [inline pages.fr/common/cmake.md:6]: Please use présent de l'indicatif
```suggestion
- Génére une recette de construction `CMakeLists.txt` depuis le répertoire d'un projet :
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.fr/common/cmake.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`cmake {{chemin/vers/le/répertoire_du_projet}}`
+
+- Générer une recette de construction, en définissant le type de construction à `Release` à l'aide d'une variable CMake :
```
Comment (Neluji) [inline pages.fr/common/cmake.md:10]:
```suggestion
- Génère une recette de construction, en définissant le type de construction à `Release` à l'aide d'une variable CMake :
```

File: pages.fr/common/cmake.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`cmake {{chemin/vers/le/répertoire_du_projet}} -D {{CMAKE_BUILD_TYPE=Release}}`
+
+- Utiliser une recette déjà générée dans un répertoire donné pour construire les artefacts :
```
Comment (Neluji) [inline pages.fr/common/cmake.md:14]:
```suggestion
- Utilise une recette déjà générée dans un répertoire donné pour construire les artefacts :
```

File: pages.fr/common/cmake.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`cmake --build {{chemin/vers/le/répertoire_de_construction}}`
+
+- Installer les artefacts de construction dans `/usr/local/` et retirer les symboles de débogage :
```
Comment (Neluji) [inline pages.fr/common/cmake.md:18]:
```suggestion
- Installe les artefacts de construction dans `/usr/local/` et retirer les symboles de débogage :
```

File: pages.fr/common/cmake.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`cmake --install {{chemin/vers/le/répertoire_de_construction}} --strip`
+
+- Installer les artefacts de construction en utilisant un préfixe personnalisé pour les chemins :
```
Comment (Neluji) [inline pages.fr/common/cmake.md:22]:
```suggestion
- Installe les artefacts de construction en utilisant un préfixe personnalisé pour les chemins :
```

File: pages.fr/common/cmake.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`cmake --install {{chemin/vers/le/répertoire_de_construction}} --strip --prefix {{chemin/vers/le/répertoire}}`
+
+- Lancer une cible de construction personnalisée :
```
Comment (Neluji) [inline pages.fr/common/cmake.md:26]:
```suggestion
- Lance une cible de construction personnalisée :
```

File: pages.fr/common/cmake.md (hunk 2 above)
Comment (Neluji) [inline pages.fr/common/cmake.md:10]:
```suggestion
- Génère une recette de construction, en définissant le type de construction à `Release` à l'aide d'une variable CMake :
```

File: pages.fr/common/cmake.md (hunk 1 above)
Comment (Neluji) [reply pages.fr/common/cmake.md:6]:
```suggestion
- Génère une recette de construction `CMakeLists.txt` depuis le répertoire d'un projet :
```

**Review by navarroaxel — APPROVED** [review]

File: pages.fr/common/cmake.md (hunk 1 above)
Comment (Neluji) [reply pages.fr/common/cmake.md]: Sorry I kind of mixed up infinitive/imperative (yeah you already told me about this 😅), but it is fixed now

Comment (navarroaxel) [reply pages.fr/common/cmake.md]: Don't worry. :+1:

**Review by marchersimon — APPROVED** [review]
Thanks @Neluji

**Pull request merged**